src/main/java/com/facebook/openwifirrm/optimizers/tpc/MeasurementBasedApApTPC.java

```diff
@@ -130,7 +130,7 @@ protected static int getCurrentTxPower(JsonArray latestDeviceStatus, String band
 				return radioObject.get("tx-power").getAsInt();
 			}
 		}
-		return 0;
+		return -1;
```

Comment:
I would prefer `null` to a magic number like `-1` when checking if the function was able to give a feasible return value.

Reply:
It returns an int, so null wouldn't work I think, but I can do an `Optional`

Reply:
I'm good with either `Optional` or `Integer`.